autoload/vital/__vital__/System/Job.vim

```diff
@@ -81,6 +84,10 @@ else
     return job
   endfunction
 
+  function! s:is_available() abort
+    return has('job') && has('patch-8.0.0027')
```

Comment:
v8.0.0027 is required only for `Job.wait()` so please change the function name to more specific to `Job.wait()` (`is_wait_available()` or whatever)